MMM-CECControlNodeHelper.js

```diff
@@ -63,14 +63,14 @@ exports.MMMCECControlNodeHelper = {
 
   turnOnCEC: function (callback) {
     this.execWrapper(
-      'echo "on 0" | cec-client ' + this.config.comport + ' -s -d 1',
+      '/opt/vc/bin/tvservice -p && sudo chvt 6 && sudo chvt 7',
```

Comment:
Is sudo needed? I ask because i think this will not work if you not run MagicMirror without Admin privilege.

Reply:
Sudo is not needed! Sorry I will remove it